accounts/abi/abi_test.go

```diff
@@ -400,9 +400,9 @@ func TestInputVariableInputLength(t *testing.T) {
 	length2[31] = byte(len(str2))
 	value2 := common.RightPadBytes([]byte(str2), 32)
 
-	exp2 := append(offset1, offset2...)
-	exp2 = append(exp2, append(length1, value1...)...)
-	exp2 = append(exp2, append(length2, value2...)...)
+	exp2 := append(offset1, offset2...)                // nozero
```

Comment:
usually comment looks like: `nolint:makezero` otherwise hard to understand what is this comment about